getting-started/angular/README.MD

```diff
@@ -1 +1 @@
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/angular/devfile.yaml)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/angular/devfile.yaml)
```

Comment:
verified workspace startup and LS basic functionality - OK
Build command is failing on `chunk asset optimization TerserPluginKilled`
![Screenshot from 2021-02-22 13-31-59](https://user-images.githubusercontent.com/29278800/108709017-794eca80-7512-11eb-8f5e-16cdbad600d8.png)